dev/TestMailer.php

```diff
@@ -4,14 +4,14 @@
  * @subpackage email
  */
 class TestMailer extends Mailer {
-	protected $emailsSent = array();
+	protected static $emailsSent = array();
```

Comment:
This is going to be problematic if running across multiple tests... creating and re-setting a new TestMailer won't flush all old emails.

I suggest keeping these as non-static if you can.